hoomd/hpmc/pytest/test_faceted_ellipsoid_union.py

```diff
@@ -0,0 +1,591 @@
+                                             (1, 0, 0, 0),
+                                             (1, 0, 0, 0),
+                                             (1, 0, 0, 1)],
+                            'overlap': [1, 1, 0, 0],
+                            'capacity': 3,
+                            'ignore_statistics': 0}
+
+faceted_ell_union_args19 = {'shapes': [faceted_ell_args_1,
+                                       faceted_ell_args_2,
+                                       faceted_ell_args_4,
+                                       faceted_ell_args_5],
+                            'positions': [(1, 1, 0),
+                                          (1, 0, 1),
+                                          (0, 0, 0),
+                                          (1, 1, 1)],
+                            'orientations': [(1, 0, 1, 0),
+                                             (1, 0, 0, 0),
+                                             (1, 0, 0, 0),
+                                             (1, 0, 1, 0)],
+                            'overlap': [1, 0, 1, 1],
+                            'capacity': 2,
+                            'ignore_statistics': 1}
+
+faceted_ell_union_args20 = {'shapes': [faceted_ell_args_1,
+                                       faceted_ell_args_3,
+                                       faceted_ell_args_4,
+                                       faceted_ell_args_5],
+                            'positions': [(1, 1, 1),
+                                          (0, 0, 1),
+                                          (0, 0, 0),
+                                          (1, 1, 0)],
+                            'orientations': [(1, 0, 0, 1),
+                                             (1, 0, 0, 0),
+                                             (1, 0, 0, 0),
+                                             (1, 1, 0, 0)],
+                            'overlap': [0, 1, 1, 0],
+                            'capacity': 4,
+                            'ignore_statistics': 0}
+
+faceted_ell_union_args21 = {'shapes': [faceted_ell_args_2,
+                                       faceted_ell_args_3,
+                                       faceted_ell_args_4,
+                                       faceted_ell_args_5],
+                            'positions': [(0, 0, 0),
+                                          (0, 1, 1),
+                                          (1, 1, 1),
+                                          (1, 0, 0)],
+                            'orientations': [(1, 0, 0, 0),
+                                             (1, 1, 0, 0),
+                                             (1, 0, 0, 1),
+                                             (1, 0, 0, 0)],
+                            'overlap': [1, 0, 0, 0],
+                            'capacity': 4,
+                            'ignore_statistics': 1}
+
+faceted_ell_union_args22 = {'shapes': [faceted_ell_args_1,
```

Comment:
Do we really need all these combinations? Some of these seem redundant. If we do, could we perhaps programmatically generate them since they seem to just be different combinations of the same building blocks?

Reply:
Removed some combinations